tests/run_tests.sh

```diff
@@ -12,8 +12,26 @@ function cleanup {
 trap cleanup EXIT
 
 # Check that we have getfattr / setfattr
-getfattr --version > /dev/null
-setfattr --version > /dev/null
+if command -v getfattr > /dev/null 2>&1
+then
+	# we have getfattr, make sure we have setfattr
+	if ! command -v setfattr > /dev/null 2>&1
+	then
+		echo "error: getfattr exists but not setfattr"
+		exit 1
+	else
+		ATTRVER="linux"
+	fi
+else
+	# no getfattr, see if we have xattr
+	if command -v xattr > /dev/null 2>&1
+	then
+		ATTRVER="macos"
+	else
+		echo "error: no getfattr or xattr available"
+		exit 1
+	fi
+fi
```

Comment:
Just delete the "have gefattr / setfattr" check without replacement, it's not worth the hassle